elevate-cpanel

```diff
@@ -6166,6 +6167,67 @@ EOS
 
 }    # --- END lib/Elevate/Components/Panopta.pm
 
+{    # --- BEGIN lib/Elevate/Components/PackageRestore.pm
+
+    package Elevate::Components::PackageRestore;
+
+    use cPstrict;
+
+    use Elevate::StageFile ();
+
+    use Cpanel::Pkgr ();
```

Comment:
Stylistic - I always try and alphabetize use statements. Is there a reason there is a space between the statements here?